packages/gatsby/src/internal-plugins/internal-data-bridge/gatsby-node.js

```diff
@@ -170,3 +170,65 @@ emitter.on(`DELETE_PAGE`, action => {
   const node = getNode(nodeId)
```

Comment:
While not necessary I think it's a good idea to add typedefs to internal plugins, since there's no need to infer those types on every run. Also, we should rethink the status of the File type, which we currently treat as if it were internal (e.g. we auto-infer File fields), but rely on a not-internal plugin to handle.